src/main/java/io/github/thebusybiscuit/slimefun4/utils/itemstack/RadiationSymptom.java

```diff
@@ -0,0 +1,50 @@
+package io.github.thebusybiscuit.slimefun4.utils.itemstack;
+
+
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import org.bukkit.entity.Player;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+
+import javax.annotation.Nonnull;
+
+public enum RadiationSymptom {
+    /**
+     * An enum of potential radiation symptoms.
+     */
```

Comment:
javadocs go above the declaration.

Also, this enum was very confusing at first, it really needs a better explanation, I didn't get that it advances progressively through these symptoms when the exposure fills up for example, that really needs much more thorough documentation...

```suggestion
/**
* An enum of potential radiation symptoms.
*
* @author Semisol
*/
public enum RadiationSymptom {
```